Update to 0.3.0 and enable phrases and substring (#4)
IronCoreLabs/try-cloaked-search: README.md

### Sample Queries

> This is a couple examples of simple term queries:

Reviewer: ```suggestion
Here are some examples of simple term queries:
```

> This is a couple examples of simple term queries:

Reviewer 2: ```suggestion
These are a couple examples of simple term queries:
```

curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:8675/try_cloaked_search/_search | jq
```

Term queries can also be combined with ORs or ANDs like this:

> 

Reviewer: I wonder if it would be better to combine the search terms into a single subquery?

Author: In this case I'm not sure it makes much of a difference. I think it'd be nice to continue to expand this section to show more complex queries.
